net/bfdd/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2019 Lucian Cristian <lucian.cristian@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bfdd
+PKG_SOURCE_DATE:=2019-08-20
+PKG_RELEASE:=1
+
+PKG_SOURCE_VERSION:=69eb45bfed9ab382a8a7bef6703a58c2624906b3
+PKG_SOURCE_URL:=https://codeload.github.com/rzalamena/bfdd/tar.gz/$(PKG_SOURCE_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_DATE).tar.gz
+PKG_HASH:=54285d84f9be301c1e50eafde8efc5e76455c66e0ad6187fc9ba35bcd76518b4
+
+PKG_MAINTAINER:=Lucian Cristian <lucian.cristian@gmail.com>
+PKG_LICENSE:=GPL-1.0-only
```

Comment:
This one is not correct according to the files in their repository.

Reply:
sorry I only see GPL, please give me a hint

Reply:
files say: either version 2 of the License, or (at your option)
 * any later version.